crates/router/src/routes/admin.rs

```diff
@@ -41,6 +41,42 @@ pub async fn merchant_account_create(
     ))
     .await
 }
+
+#[cfg(feature = "v2")]
+/// Merchant Account - Create
+///
+/// Create a new account for a merchant and the merchant could be a seller or retailer or client who likes to receive and send payments.
+#[utoipa::path(
+    post,
+    path = "/v2/accounts",
+    request_body= MerchantAccountCreateV2,
+    responses(
+        (status = 200, description = "Merchant Account Created", body = MerchantAccountResponseV2),
+        (status = 400, description = "Invalid data")
+    ),
+    tag = "Merchant Account",
+    operation_id = "Create a Merchant Account",
+    security(("admin_api_key" = []))
+)]
```

Comment:
We can remove the OpenAPI documentation snippets, since they are all in a separate crate?